### PR TITLE
minor readability fixes, minor issue resolution

### PR DESCRIPTION
**Pull request by goi42**
a few more minor changes.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
...
 and we welcome contributions of all kinds:
 
-* New lessons;
-* Fixes to existing material;
-* Bug reports; and
-* Reviews of proposed changes.
+* New lessons
+* Fixes to existing material
+* Bug reports
+* Reviews of proposed changes
```
Comment (alexpearce) [inline CONTRIBUTING.md]: You don't like this style of writing lists?

Comment (goi42) [reply CONTRIBUTING.md]: No--the connection between the thoughts is implied by the bullets, making the extra punctuation and 'and' redundant.

**Comment by vlisovsk** [comment]
I didn't quite get the changes in the second-analysis-steps: why did you changed the path to Particles as if we were using mDST?

File: CONTRIBUTING.md (hunk 1 above)
Comment (alexpearce) [reply CONTRIBUTING.md]: OK. This is a stylistic choice that I disagree with, but don't hold the opinion very strongly. Either way, we should try to be consistent with the formatting in other lists elsehwere.

**Comment by goi42** [comment]
@vlisovsk those changes were made at the request of a user who said the instructions were broken. I thought they'd already been committed in a previous pull request...

File: CONTRIBUTING.md (hunk 1 above)
Comment (goi42) [reply CONTRIBUTING.md]: I believe this is the only list that had the `; and` construction, though I could be wrong. In any case, I agree we should be consistent.

**Comment by vlisovsk** [comment]
Ah, this is that story with restripping, sorry.

**Comment by alexpearce** [comment]
Mm indeed, I thought the idea was to leave the "Easy" issues for the students to fix on Friday.

Also, are all the whitespace changes necessary? It makes `git blame` a lot less useful. I guess your editor is configured to remove them, but I'd prefer to keep them if the usage is the majority in a given file.

**Comment by roelaaij** [comment]
I would go so far as to suggest to remove 0074aea, so it can be fixed by a student; it could be left on a branch so it can be cherry picked back in if not fixed on Friday. I'll reopen #56 if you agree.

**Comment by goi42** [comment]
ah I'm sorry--I forgot we were finding issues for the students. I updated these because they are related to changes I'd made to the repo haha. I'll defer to the higher powers on whether to keep these.

**Comment by goi42** [comment]
No, the whitespace changes aren't necessary. Yes, they're automatically removed by my editor.

**Comment by alexpearce** [comment]
There is no higher power than a contributor 😄 hopefully everyone in the lhcb org has merge rights on this repo.

How about this: seeing as nothing here is critical for the teaching that's left, we leave this PR open, and rebase and merge it after the students have finished contributing on Friday. What do you think?

**Comment by goi42** [comment]
My thoughts exactly.

**Pull request closed without merging**